sklearn/multioutput.py

```diff
@@ -150,3 +156,134 @@ def score(self, X, y, sample_weight=None):
+
+    def __init__(self, estimator, n_jobs=1):
+
+        self.estimator = estimator
+        self.n_jobs = n_jobs
+
+    def fit(self, X, y):
+        """ Fit the model to data.
+        Fits a seperate model for each output variable.
+        Parameters
+        ----------
+        X : (sparse) array-like, shape = [n_samples, n_features]
+            Data.
+        y : (sparse) array-like, shape = [n_samples, n_outputs]
+            Multi-output targets. An indicator matrix turns on multilabel
+            classification.
+        Returns
+        -------
+        self
+        """
+
+        _check_estimator(self.estimator)
+        # check to see that the X and y have the same number of rows.
+        check_consistent_length(X, y)
+        if y.ndim == 1:
+            raise ValueError("Please use the base estimator directly"
+                             "for single output classification")
+
+        self.n_outputs_ = y.shape[1]
+        self.estimators_ = Parallel(n_jobs=self.n_jobs)(delayed(_fit_classification)(
+            self.estimator, X, y[:, i]) for i in range(self.n_outputs_))
+        return self
+
+    def predict(self, X):
+        """Predict multi-output variable using a model
+         trained for each target variable.
+        Parameters
+        ----------
+        X : (sparse) array-like, shape = [n_samples, n_features]
+            Data.
+        Returns
+        -------
+        y : (sparse) array-like, shape = [n_samples, n_outputs]
+            Multi-output targets predicted across multiple predictors.
+            Note:  Separate models are generated for each predictor.
+        """
+        check_is_fitted(self, 'estimators_')
+
+        results = np.column_stack([estimator.predict(X) for estimator in
+                                   self.estimators_])
+        return results
+
+    def predict_proba(self, X):
+        """Probability estimates. This returns prediction probabilites
+        for each class for each label.
+        Parameters
+        ----------
+        X : array-like, shape = [n_samples, n_features]
+        Returns
+        -------
+        T : (sparse) array-like, shape = [n_samples, n_classes, n_outputs]
+            The class probabilities of the samples for each of the outputs
+        """
+        check_is_fitted(self, 'estimators_')
+
+        results = np.dstack([estimator.predict_proba(X) for estimator in
+                            self.estimators_])
+        return results
+        
+    def score(self, X, y):
+        """"Returns the mean accuracy on the given test data and labels.
+        Parameters
+        ----------
+        X : array-like, shape = [n_samples, n_features]
+        y : array-like, shape = [n_samples, n_outputs]
+        Returns
+        -------
+        scores : array-like, shape = [n_outputs, 1]
+            Array of p_target floats of the mean accuracy
+            of each estimator_.predict wrt. y.
+        """
+        check_is_fitted(self, 'estimators_')
```

Comment:
You should write your own implementation over here. Basically, for a given sample, if all the outputs of `y_true` and `y_pred` are the same, you should give a score of 1 and 0 otherwise.